### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:

Today I tried the kfervice python related features and reviewed some documents, and hit some small problems. The PR is going to fix the problems. Thanks.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/267)

### CODE REVIEW

File: python/pytorchserver/README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ An empty result will indicate success.
 
 You can build and publish your own image for development needs. Please ensure that you modify the kfservice files for PyTorch in the api directory to point to your own image.
 
-To build your own image, run
+To build your own image, navigate up one directory level and run:
```
Comment (animeshsingh) [inline python/pytorchserver/README.md]: Make sure to specify which explicitly which director you are in

File: python/sklearnserver/README.md [hunk 2]
```diff
@@ -107,12 +107,12 @@ An empty result will indicate success.
 
 ## Building your own Scikit-Learn Server Docker Image
 
-You can build and publish your own image for development needs. Please ensure that you modify the kfservice files for XGBoost in the api directory to point to your own image.
+You can build and publish your own image for development needs. Please ensure that you modify the kfservice files for Scikit-Learn in the api directory to point to your own image.
 
-To build your own image, run
+To build your own image, navigate up one directory level and run:
```
Comment (animeshsingh) [inline python/sklearnserver/README.md]: Same as above

File: python/xgbserver/README.md [hunk 3]
```diff
@@ -112,10 +112,10 @@ An empty result will indicate success.
 
 You can build and publish your own image for development needs. Please ensure that you modify the kfservice files for XGBoost in the api directory to point to your own image.
 
-To build your own image, run
+To build your own image, navigate up one directory level and run:
 
 ```bash
-docker build -t $USER/xgbserver -f xgb.Dockerfile .
+docker build -t docker_user_name/xgbserver -f xgb.Dockerfile .
```
Comment (animeshsingh) [inline python/xgbserver/README.md]: Same as above

**Comment by jinchihe** [comment]
@animeshsingh Done. Updated to point to `python` directory, since the Dockerfile is in the upper one directory of the README :-)

**Pull request merged**